lib/reference/keywords.mjs

```diff
@@ -286,6 +286,16 @@ export const camelCaseKeywords = new Set([
 
 export const keyframeSelectorKeywords = new Set(['from', 'to']);
 
+// https://drafts.csswg.org/scroll-animations-1/#view-progress-timelines
+export const namedTimelineRangeKeywords = new Set([
```

Comment:
Can you export this set in `keyframe-selector-notation/index.mjs` and replace

https://github.com/stylelint/stylelint/blob/e7e54941b9376bc63d5c7343c1d1ef56eb0ab04c/lib/rules/keyframe-selector-notation/index.mjs#L20

Reply:
I think we best do that in a separate PR.

The list used by `keyframe-selector-notation` is either wrong or outdated.
`enter` does not exist (anymore) and `entry-crossing`, `exit-crossing` do exist.

Fixing that is out of scope.

Reply:
We will have to wait for this PR to be merged then.